File: wms_extension/multimodal-management-system/mms/ruoyi-ui/src/api/algorithm/mrpinfo.js
import request from '@/utils/request'

export function listMrpinfo(query) {
  return request({
    url: '/algorithm/mrpinfo/list',
    method: 'get',
    params: query
  })
}

export function getMrpinfo(itemCode) {
  return request({
    url: '/algorithm/mrpinfo/' + itemCode,
    method: 'get'
  })
}

export function addMrpinfo(data) {
  return request({
    url: '/algorithm/mrpinfo',
    method: 'post',
    data: data
  })
}

export function updateMrpinfo(data) {
  return request({
    url: '/algorithm/mrpinfo',
    method: 'put',
    data: data
  })
}

export function delMrpinfo(itemCode) {
  return request({
    url: '/algorithm/mrpinfo/' + itemCode,
    method: 'delete'
  })
}

File: wms_extension/multimodal-management-system/Algorithm/algorithm/src/main/java/com/algorithm/algorithm/entity/MRPInfo.java
package com.algorithm.algorithm.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 功能：mrp信息
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class MRPInfo {
    String itemCode;
    double averageAvailability;
}
